rasr/feature_scorer.py

```diff
@@ -126,25 +129,54 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        *,
+        mixtures: Union[str, tk.Path],
+        model: Union[str, tk.Path],
+        io_map: Dict[str, str],
+        label_log_posterior_scale: float = 1.0,
+        label_prior_scale: float = 0.7,
+        label_log_prior_file: Union[str, tk.Path] = None,
```

Comment:
Wouldn't we want the parameters to be called the same as in other feature scorers and in the rasr config itself? I.e.  `mixtures` -> `prior_mixtures`, `label_prior_scale` -> `priori_scale` and `label_log_prior_file` -> `prior_file`.

Reply:
No, I explicitly explained above that I would like to make the variables more expressive and correct, by decoupling it from the old GMM based feature scorers

Reply:
Okay, I didn't see the conversation. If other people also agree then this renaming is fine for me.